tests/Unit/DesignationServiceTest.php

```diff
@@ -51,22 +51,60 @@ public function test_job_handles_data_correctly()
             'name' => $request['name']
         ]);
     }
+    public function test_service_to_delete_designation_successfully(): void
+    {
+        $designation = Designation::factory()->create();
+        $service = new DesignationService();
+
+        $this->assertInstanceOf(Designation::class, $designation);
+        $result = $service->deleteDesignation($designation->id);
+
+        $this->assertDatabaseMissing('designations', ['id' => $designation->id]);
+        $this->assertTrue($result);
 
-    public function test_designation_not_created_invalid_updated_at()
+    }
+    public function test_service_to_delete_designation_not_found(): void
     {
+        $service = new DesignationService();
+        $result = $service->deleteDesignation(9999);
+        $this->assertFalse($result);
+    }
+    public function test_for_job_to_successfully_delete_designation(): void
+    {
```

Comment:
this Queue test should be on its own, more like a separate file

Reply:
done